### PR TITLE
PSMDB-1234 Release notes 6.0.5-4

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	modified:   docs/comparison.md
	modified:   docs/glossary.md
	modified:   docs/index.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml
new file: docs/release_notes/6.0.5-4.md

### CODE REVIEW

**Review by igorsol — APPROVED** [review]
LGTM

File: docs/release_notes/6.0.5-4.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Percona Server for MongoDB 6.0.5-4 (2023-03-29)
+
+| Release date | March 29, 2023|
+|------------- | ---------------|
+| **Installation** | [Installing Percona Backup for MongoDB](../install/index.md) |
+
+
+Percona Server for MongoDB 6.0.5-4 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.4 Community Edition.
```
Comment (sandraromanchenko) [inline docs/release_notes/6.0.5-4.md]:
```suggestion
fully-compatible, drop-in replacement for MongoDB 6.0.5 Community Edition.
```

File: docs/release_notes/6.0.5-4.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Percona Server for MongoDB 6.0.5-4 (2023-03-29)
+
+| Release date | March 29, 2023|
+|------------- | ---------------|
+| **Installation** | [Installing Percona Backup for MongoDB](../install/index.md) |
```
Comment (sandraromanchenko) [inline docs/release_notes/6.0.5-4.md]:
```suggestion
| **Installation** | [Installing Percona Server for MongoDB](../install/index.md) |
```
BTW RN for prev versions should also be fixed

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by janwieremjewicz — APPROVED** [review]

**Comment by kwilktx** [comment]
@nastena1606 Found a reference to v6.0.4 in the new 6.0.5 release notes.

Please update. the description:  https://psmdb-docs-6-0-pr-684.onrender.com/release_notes/6.0.5-4.html

Change this: 
"Percona Server for MongoDB 6.0.5-4 is an enhanced, source-available, and highly-scalable database that is a fully-compatible, drop-in replacement for MongoDB 6.0.4 Community Edition."

to this:
"Percona Server for MongoDB 6.0.5-4 is an enhanced, source-available, and highly-scalable database that is a fully-compatible, drop-in replacement for MongoDB 6.0.5 Community Edition."

Still checking through the bugs, etc. Will let you know more in a bit

**Comment by kwilktx** [comment]
Also under Release Highlights - would be good to immediately highlight that we ADDED support for AWS IAM

Release Highlights:

-"Added support for authentication using ..."

**Review by kwilktx — CHANGES_REQUESTED** [review]
Add the fix for these following bugs as they are ones that customers might likely run into:"


https://jira.mongodb.org/browse/SERVER-73266

Fixes deadlock that can occur during index creation


https://jira.mongodb.org/browse/SERVER-73009
Resolved problem in v6.0.x  version on clustered collections where returned results in increasing instead of the decreasing order requested

https://jira.mongodb.org/browse/SERVER-72512


https://jira.mongodb.org/browse/SERVER-71219

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**